login.html

```diff
@@ -4,41 +4,39 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="icon" href="/assets/favicon.ico">
-    <link rel="stylesheet" href="css/reset.css" />
-    <link rel="stylesheet" href="css/palette.css" />
     <link
-      rel="stylesheet"
-      as="style"
-      crossorigin
-      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/variable/pretendardvariable.min.css"
+    rel="stylesheet"
+    as="style"
+    crossorigin
+    href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/variable/pretendardvariable.min.css"
     />
-    <script src="https://kit.fontawesome.com/5b35a6d114.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="css/reset.css" />
     <link rel="stylesheet" href="css/style.css" />
     <link rel="stylesheet" href="css/auth.css" />
     <title>판다마켓</title>
   </head>
 <body>
   <main>
     <div class="logo-auth-box">
-      <h1>
-        <a href="/">
-          <img src="assets/auth_logo.png" alt="판다마켓"/>
-        </a>
-      </h1>
+      <a href="/">
+        <img src="assets/auth_logo.png" alt="판다마켓"/>
+      </a>
     </div>
     <div class="cont-area">
-      <div class="group-inp">
-        <label for="inputEmail">이메일</label>
-        <input type="email" id="inputEmail" placeholder="codeit@email.com">
-      </div>
-      <div class="group-inp">
-        <label for="inputPwd">비밀번호</label>
-        <div class="inputPwd-wrap">
-          <input type="password" id="inputPwd" placeholder="비밀번호를 입력하세요">
-          <i class="fa-regular fa-eye-slash"></i>
+      <form action="" method="post">
+        <div class="group-input">
+          <label for="inputEmail">이메일</label>
+          <input type="email" class="input-font" id="inputEmail" placeholder="이메일을 입력하세요">
         </div>
-      </div>
-      <button type="submit" class="btn login-btn">로그인</button>
+        <div class="group-input">
+          <label for="inputPwd">비밀번호</label>
+          <div class="inputPwd-wrap">
+            <input type="password" class="input-font" id="inputPwd" placeholder="비밀번호를 입력하세요">
+            <img src="assets/visibility_off.png" alt="비밀번호 숨김" class="toggle-password"/>
```

Comment:
❗️ 수정요청  
추후 구현하시겠지만 해당 요소는 비밀번호가 보이고 안 보이게 만드는 상호작용 요소이므로 button요소로 작성하시는 것을 추천드려요.
```suggestion
            <button type="button"><img src="assets/visibility_off.png" alt="비밀번호 숨김" class="toggle-password"/></button>
```